packages/web/components/Card/Share/__tests__/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { render, fireEvent } from 'test-utils';
+import Share from '..';
+
+describe('ShareModal component', () => {
+	it('should open the sharing modal when clicking', () => {
+		window.open = jest.fn();
+		document.execCommand = jest.fn();
+
+		const { container, getByTestId } = render(<Share />);
+
+		const shareButton = getByTestId('share');
```

Comment:
```suggestion
		const shareButton = container.firstChild;
```

Reply:
Eu discordo dessa mudança, assim o teste fica dependente da implementação. Se em algum momento o "share" não for mais um filho direto, o teste vai falhar sem necessariamente a funcionalidade ter quebrado.